kloppy/_providers/wyscout.py

```diff
@@ -23,16 +26,30 @@ def load(
         event_types:
         coordinates:
         event_factory:
+        data_model_version:
     """
-    deserializer = WyscoutDeserializer(
-        event_types=event_types,
-        coordinate_system=coordinates,
-        event_factory=event_factory or get_config("event_factory"),
-    )
-    with open_as_file(event_data) as event_data_fp:
-        return deserializer.deserialize(
-            inputs=WyscoutInputs(event_data=event_data_fp),
+    if not data_model_version:
```

Comment:
I'm wondering whether we can somewhat harden the implementation of this function. I see two potential issues with the current implementation.

1. The function won't do anything when a user provides a string that is neither "V3" nor "V2" for the `data_model_version` parameter. This issue could be avoided by calling the `identify_data_model_version` function if a user provides an argument for the `data_model_version` parameter that is neither "V3" nor "V2".
2. The `if` and `elif` branches contain a lot of duplicated code. The only difference between the two branches is the name of the deserializer class. This issue could be avoided by storing the class in the `deserializer` variable. In addition, the `identify_data_model_version` function could also return the appropriate class.